cmd/manager/main.go

```diff
@@ -215,10 +217,9 @@ func main() {
 			cmd.FatalError(setupLogger, err, "unable to create controller", "name", controllers.NodeKernelClusterClaimReconcilerName)
 		}
 	} else {
-		signImage := cmd.GetEnvOrFatalError("RELATED_IMAGE_SIGN", setupLogger)
-		buildSignAPI := buildsignocpbuild.NewManager(client, buildSignCombinerAPI, kernelOsDtkMapping, signImage, scheme)
+		builSignAPI := buildsign.NewManager(client, resourceManager, scheme)
```

Comment:
typo: builSignAPI ---> buildSignAPI

Reply:
It is done on purpose. Typo comes from u/s.